src/elements/content-uploader/ItemRemove.js.flow

```diff
@@ -1,27 +1,22 @@
-/**
- * @flow
- * @file Item remove component
- */
-
 import * as React from 'react';
-import { injectIntl } from 'react-intl';
-import type { IntlShape } from 'react-intl';
-import PlainButton from '../../components/plain-button/PlainButton';
-import Tooltip from '../../components/tooltip';
-import IconClose from '../../icons/general/IconClose';
-import messages from '../common/messages';
-import { STATUS_ERROR, STATUS_IN_PROGRESS, STATUS_STAGED } from '../../constants';
+import { useIntl } from 'react-intl';
+import { IconButton, Tooltip } from '@box/blueprint-web';
+import { BlackOpacity80, Size5 } from '@box/blueprint-web-assets/tokens/tokens';
```

Comment:
```suggestion
import { GrayBlack, Size5 } from '@box/blueprint-web-assets/tokens/tokens';
```
I think `GrayBlack` is probably what we want because its should be using `#000000` unless you saw an `alpha` being used